src/containers/my-offers/my-offers-card/MyOffersCard.styles.ts

```diff
@@ -0,0 +1,47 @@
+const ellipsisTextStyle = (linesCount: number) => ({
+  display: '-webkit-box',
+  WebkitLineClamp: linesCount,
+  lineClamp: linesCount,
+  WebkitBoxOrient: 'vertical',
+  boxOrient: 'vertical',
+  overflow: 'hidden'
+})
+
+export const styles = {
+  container: {
+    width: '100%',
+    minHeigth: '400px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '16px',
+    justifyContent: 'space-between'
+  },
+  title: {
+    ...ellipsisTextStyle(2),
+    typography: 'midTitle',
```

Comment:
change all typographies in styles to enums

Reply:
fixed